soap/src/java/com/zimbra/soap/mail/message/GetContactFrequencyResponse.java

```diff
@@ -0,0 +1,25 @@
+package com.zimbra.soap.mail.message;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlElements;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import com.zimbra.common.soap.MailConstants;
+import com.zimbra.soap.mail.type.ContactFrequencyData;
+
+@XmlRootElement(name=MailConstants.E_GET_CONTACT_FREQUENCY_RESPONSE)
+public class GetContactFrequencyResponse {
+
+    public GetContactFrequencyResponse() {}
+
+    @XmlElements({
```

Comment:
If there is only one class, don't need `@XmlElements`.  Suspect this makes the WSDL un-necessarily complicated - you probably end up with list of elements which can be a choice from the set [<data>].  If you just use `@XmlElement` with a `List` then the WSDL will allow a list of "<data>" elements.

Reply:
Ah OK, thanks!